WcaOnRails/app/models/competition.rb

```diff
@@ -907,6 +908,14 @@ def guests_entry_fee_required?
     )
   end
 
+  def all_guests_allowed?
+    free_entry_text == "anyone"
```

Comment:
Try to define these as enum. You can take `stripe_charge.rb` as an example.

Reply:
This is an article I found helpful when researching about Rails enums: https://betterprogramming.pub/how-to-use-enums-in-rails-6-87600e292476

Reply:
Enums! Haven't used them in so long that I forgot they exist.